rdmo/management/rules.py

```diff
@@ -144,6 +152,8 @@ def is_legacy_reviewer(user) -> bool:
 rules.add_perm('tasks.add_task_object', is_element_editor)
 rules.add_perm('tasks.change_task_object', is_element_editor)
 rules.add_perm('tasks.delete_task_object', is_element_editor)
+# toggle current site field perm
+rules.add_perm('tasks.change_task_toggle_own_site', is_element_editor | is_editor_for_current_site)
```

Comment:
I would skip the `_own_` and use just `toogle_site`. I wonder if we need `is_element_editor` here. This would mean that an editor sees this on a site where they are not editor and removes it from this site. But ok they can do it from their own site anyway.

Reply:
yes, renamed and removed the `is_element_editor` from here